cpu/sam3/include/periph_cpu.h

```diff
@@ -30,6 +30,7 @@ extern "C" {
 
 /**
  * @brief Overwrite the default gpio_t type definition
```

Comment:
If I recall correctly, grouping only works with `@name` instead of `@brief`

Reply:
Yepp.

Reply:
Removed the doc in https://github.com/RIOT-OS/RIOT/pull/16813/commits/fab0976f73ef292fd020e3832274067f3d845382 in favor of a `#ifndef` block and also fixed another appearance of `@brief` and `@` in https://github.com/RIOT-OS/RIOT/pull/16813/commits/35b6d797125c3c202c4175d81e841a6e22474c55.